docs/README.md

```diff
@@ -149,39 +160,75 @@ redirect_from:
 - [Custom map settings](./configuring-metabase/custom-maps.md)
 - [Customizing the Metabase Jetty Webserver](./configuring-metabase/customizing-jetty-webserver.md)
 
-## Usage and performance tools
+### Usage and performance tools
 
 - [Overview](./usage-and-performance-tools/start.md)
 - [Auditing tools](./usage-and-performance-tools/audit.md)
 - [Tracking query errors](./usage-and-performance-tools/tools.md)
 
-## Metabase API
+### Metabase API
 
 - [API reference](./api-documentation.md)
 - [API tutorial](https://www.metabase.com/learn/administration/metabase-api)
 
-## Premium features
+### Troubleshooting
 
-The Enterprise and Pro plans of Metabase offer additional features.
+- [Get unstuck](./troubleshooting-guide/index.md)
 
-- [Overview](./paid-features/start.md)
-- [Getting and activating the Enterprise edition](./paid-features/activating-the-enterprise-edition.md)
-- [List of premium features](./paid-features/overview.md)
+## Getting help
 
-## Metabase community
+### Troubleshooting
 
+- [Troubleshooting guides](troubleshooting-guide/index.md)
 - [Metabase forum](https://discourse.metabase.com/)
-- [Community stories](https://www.metabase.com/community)
-- [Case studies](https://www.metabase.com/case_studies/)
-- [Metabase Blog](https://www.metabase.com/blog)
-- [Source code repository on GitHub](https://github.com/metabase/metabase)
+- [Configuring logging](./configuring-metabase/log-configuration.md)
 
-## [Developers guide](developers-guide/start.md)
+### [Tutorials and guides](https://www.metabase.com/learn)
 
-Contribute to the Metabase open source project!
+[Learn Metabase](https://www.metabase.com/learn) has a ton of articles on how to use Metabase, data best practices, and more.
+
+## More resources
+
+### [Learn Metabase](https://www.metabase.com/learn)
+
+Guides on working with data.
+
+### [Discussion](https://discourse.metabase.com)
 
-## [Metabase Cloud](https://www.metabase.com/cloud/docs)
+Share and connect with other Metabasers.
+
+### [Premium features](./paid-features/start.md)
```

Comment:
Do we want the heading to be "Paid features" to match the destination page? Or is this intentional for the sake of search? Just so I know whether to mix in premium / paid in other places too :)

Reply:
Changed to "Paid". Nice catch.